.github/settings.yml

```diff
@@ -84,10 +84,18 @@ labels:
 - name: area/security
   color: 0052cc
   description: Security relevant
+- name: area/network
+  color: 0052cc
+  description: Network related
+  oldname: area/networking
 - name: area/testing
   color: 0052cc
   description: Testing related
-
+  oldname: testing
```

Comment:
do we actually need any of those `area/` labels besides `area/ipcei`?

Reply:
one could argue that even `area/ipcei` is not relevant since (almost) everything that we do is `area/ipcei`. However, these labels are kind of an abstraction to get the topic/area of the issue/pr without reading the description. Additionally, these are metrics to get a summary on which topics we are working or have worked in the past.

IMO these benefits are worth it.

Reply:
Well, but only if you use them...which we didn't really do so far. Besides, what value would be provided by those metrics?

Reply:
> Well, but only if you use them...which we didn't really do so far.
True, but this can be changed if we see any benefit in it and commit to it.

I don't see the problem on marking an issue with an label while I am creating the issue. It only takes several seconds.

> Besides, what value would be provided by those metrics?
I guess metrics of these kinds are important if somebody wants to know what we are doing the whole day, without reading every contribution. It just gives a quick overview on which topics we are working on.
Sure, one could argue that these metrics are not valid and I am understand this. They don't represent the actual work but are only an abstract description.